### ISSUE TITLE
Strain window

### ISSUE THREAD

**Issue by chihyu722** [dicengine/dice#245]
Hi, 
I want to analyze my test piece VSG strain, but is the DICe gauge size which is equal to the strain window (window size)?
I have seen the video (DICe Demo 2D), and Speaker said that gauge size automatically increments the step size.
but, I'm not sure this define

**Comment by bvanmieg** [comment]
Dear,
Check the idics good practice manual on:

[Idics](https://www.google.com/url?sa=t&source=web&rct=j&url=https://idics.org/guide/DICGoodPracticesGuide_PrintVersion-V5h-181024.pdf&ved=2ahUKEwing5eR58n1AhUth_0HHXziDuUQFnoECAMQAQ&usg=AOvVaw0qN9Oil0BxHzjh5OQBgvWz) 

Page 75 explains how the VSG is calculated.

Regards,
Bart

**Comment by chihyu722** [comment]
Thanks for your reply

Therefore, L*window (the window size in terms of pixels) is the same as gauge size

**Comment by dicengine** [comment]
You are right, in the paper by Reu the VSG size is more like an area or strain patch. In DICe, the window size in pixels is the length of one side of the window.
